plugin.rb

```diff
@@ -14,151 +14,31 @@
 register_svg_icon "far-clipboard" if respond_to?(:register_svg_icon)
 
 after_initialize do
-  load File.expand_path("../app/jobs/onceoff/rename_canned_replies.rb", __FILE__)
-
-  module ::CannedReply
-    PLUGIN_NAME ||= "discourse-canned-replies".freeze
-    STORE_NAME ||= "replies".freeze
+  module ::DiscourseCannedReplies
+    PLUGIN_NAME ||= "discourse-canned-replies"
+    STORE_NAME ||= "replies"
```

Comment:
Can this be a simple const definition instead?

```suggestion
    PLUGIN_NAME = "discourse-canned-replies"
    STORE_NAME = "replies"
```